package.json

```diff
@@ -7,29 +7,30 @@
   "devDependencies": {
     "bower": "1.8.4",
     "geckodriver": "1.6.1",
-    "grunt": "1.0.1",
+    "grunt": "^1.0.3",
```

Comment:
I would advise a pinned version number so the build doesn't change when the upstream deps change (or rather have a new version released.) But I know opinions differ on this.

Reply:
Good point. Just updated and pinned versions in both package.json and bower.json.